src/App.js

```diff
@@ -1,16 +1,115 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList.js';
+import TodoForm from './components/TodoComponents/TodoForm.js';
+import './components/TodoComponents/Todo.css'
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      input: '',
+      tasks: [{
+          task: 'Get some sleep',
+          id: 123456789,
+          completed: false
+        },
+        {
+          task: 'Organize Garage',
+          id: 234567891,
+          completed: false
+        },
+        {
+          task: 'Bake Cookies',
+          id: 345678901,
+          completed: false
+        },
+        {
+          task: 'Finish Project',
+          id: 456789012,
+          completed: false
+        }
+      ]
+    }
+  }
+
+
+  keypressHandler = event => {
+    const obj = {
+      task: this.state.input,
+      id: Date.now(),
+      completed: false
+    }
+    if (event.key === 'Enter') {
+      this.setState({
+        tasks: [...this.state.tasks, obj],
+        input: ''
+      });
+    }
+  }
+
+  changeHandler = event => {
+    const value = event.target.value;
+    this.setState({
+      input: value,
+    })
+  }
+  clickHandler = (event, item) => {
+    event.preventDefault();
+    if (event.target.classList.contains('strikethrough')) {
```

Comment:
I have a feeling way to toggle todo's as complete or not could be done a little better and more efficient, especially with that loop. There could be some toggle method that will change the current state for single todo without need of loop.